internet/dns_utilities.py

```diff
@@ -55,6 +53,21 @@ def extract_url_base_and_extension(_url):
 
         return url_clean, url_extension
 
+    @staticmethod
+    def extract_url_base(_url):
+        pattern = re.compile(r'(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})')
+        is_match_ip_syntax = pattern.search(_url)
+        if is_match_ip_syntax:
+            url_clean = is_match_ip_syntax.group(0)
+        else:
+            try:
+                url_clean = get_fld(_url)
+            except ValueError:
+                url_clean = _url
+                Logger().warning(f'{_url} is invalid')
```

Comment:
Review the warning to be more clear
```suggestion
                Logger().warning(fDNSUtilities.extract_url_base have error: '{_url} is invalid')
```